test/e2e/cloud/gcp/cluster_upgrade.go

```diff
@@ -101,9 +101,9 @@ func masterUpgrade(f *framework.Framework, v string) error {
 	}
 }
 
-// masterUpgradeGCEWithKubeProxyDaemonSet upgrades master node on GCE with enabling/disabling the daemon set of kube-proxy.
+// instanceUpgradeGCEWithKubeProxyDaemonSet upgrades instance node on GCE with enabling/disabling the daemon set of kube-proxy.
```

Comment:
Is this upgrading a singular instance of apiserver/controller-manager/scheduler in the control plane, or is it upgrading the entire control plane?